lfroms/clouds
# Fix location search

### What is the problem being solved in this PR?
The MapKit API has changed, resulting in 0 returned results from search.

### Related issues or PRs
N/A

### Why did you choose this approach?
- Adjusting the filtering workaround so that it matches the new format.

### How to test changes
1. Start the app.
2. Search for a location. Ensure that you only get cities in Canada.

### Checklist
